ntoskrnl/mm/ARM3/mdlsup.c

```diff
@@ -1259,28 +1238,10 @@ MmProbeAndLockPages(IN PMDL Mdl,
                     Address = MiPteToAddress(PointerPte);
                     if (Address <= MM_HIGHEST_USER_ADDRESS)
                     {
-                        //
-                        // What kind of lock were we using?
-                        //
-                        if (UsePfnLock)
-                        {
-                            //
-                            // Release PFN lock
-                            //
-                            MiReleasePfnLock(OldIrql);
-                        }
-                        else
-                        {
-                            /* Release process working set */
-                            MiUnlockProcessWorkingSet(CurrentProcess, PsGetCurrentThread());
-                        }
-
                         //
                         // Access the page
                         //
-
-                        //HACK: Pass a placeholder TrapInformation so the fault handler knows we're unlocked
-                        Status = MmAccessFault(TRUE, Address, KernelMode, (PVOID)(ULONG_PTR)0xBADBADA3BADBADA3ULL);
+                        Status = MmAccessFault(TRUE, Address, KernelMode, NULL);
```

Comment:
Stupid remark, but you pass TRUE/FALSE values to an ULONG field.
Shouldn't we define a set of flags somewhere, because currently I see only a set of `MI_IS_*` set of macroses which use raw constants inside.

Reply:
Yes, I have to make up some names.